### PR TITLE
add dygraph prune quick_start

### PR DESCRIPTION
**Pull request by yukavio**

### CODE REVIEW

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+2. 构建模型和数据集
+3. 进行预训练
+4. 剪裁
+5. 训练剪裁后的模型
+
+以下章节依次次介绍每个步骤的内容。
+
+## 1. 导入依赖
+
+PaddleSlim依赖Paddle2.0-rc1版本，请确认已正确安装Paddle，然后按以下方式导入Paddle和PaddleSlim:
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:16]: 依赖Paddle2.0

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import paddle.vision.models as models
+from paddle.static import InputSpec as Input
+from paddle.vision.datasets import Cifar10
+from paddleslim.dygraph import L1NormFilterPruner
+```
+
+## 2. 构建网络和数据集
+
+该章节构造一个用于对CIFAR10数据进行分类的分类模型，选用`MobileNetV1`，并将输入大小设置为`[3, 32, 32]`，输出类别数为10。
+为了方便展示示例，我们使用Paddle提供的预定义分类模型，执行以下代码构建分类模型：
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:29]: 最好显式说明是使用的Paddle高层API，并给个文档链接：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/vision/models/mobilenetv1/MobileNetV1_cn.html#mobilenetv1

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+optimizer = paddle.optimizer.Momentum(
+        learning_rate=0.1,
+        parameters=net.parameters())
+model = paddle.Model(net, inputs, labels)
+model.prepare(
+        optimizer,
+        paddle.nn.CrossEntropyLoss(),
+        paddle.metric.Accuracy(topk=(1, 5)))
+
+val_dataset = mnist = Cifar10(mode='test')
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:44]: `mnist`没用了吧？

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+inputs = [Input([None, 3, 32, 32], 'float32', name='image')]
+labels = [Input([None, 1], 'int64', name='label')]
+optimizer = paddle.optimizer.Momentum(
+        learning_rate=0.1,
+        parameters=net.parameters())
+model = paddle.Model(net, inputs, labels)
+model.prepare(
+        optimizer,
+        paddle.nn.CrossEntropyLoss(),
+        paddle.metric.Accuracy(topk=(1, 5)))
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:42]: 说明下这段是干什么的，并给出相关API文档链接。

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+model.fit(train_dataset, epoch=2, batch_size=128, verbose=1)
+```
+
+
+## 4. 剪裁卷积层通道
+
+### 4.1 计算剪裁之前的FLOPs
+
+```
+FLOPs = paddle.flops(net, input_size=[1, 3, 32, 32], print_detail=True)
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:62]: 这里其实还有另外一个实现，可以评估下用哪个。

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+### 4.2 剪裁
+
+在这里我们先对模型结构进行敏感度分析，再根据敏感度分析的结果进行裁剪。
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:67]: 快速开始文档里就不要涉及敏感度分析了，直接上剪裁吧

File: dygraph_docs/quick_start/pruning_tutorial.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+model.evaluate(val_dataset, batch_size=128, verbose=1)
+```
+
+### 5.2 对模型进行微调
+对模型进行finetune会有助于模型恢复原有精度。
+以下代码对裁剪过后的模型进行评估后执行了一个`epoch`的微调，再对微调过后的模型重新进行评估：
+
+```
+model.fit(train_dataset, epoch=1, batch_size=128, verbose=1)
+model.evaluate(val_dataset, batch_size=128, verbose=1)
```
Comment (wanghaoshuang) [inline dygraph_docs/quick_start/pruning_tutorial.md:104]: 最后可以展示下剪裁后的模型summary， 高层API有相关接口。

**Review by wanghaoshuang — CHANGES_REQUESTED** [review]

File: dygraph_docs/quick_start/pruning_tutorial.md (hunk 1 above)
Comment (qingqing01) [reply dygraph_docs/quick_start/pruning_tutorial.md:16]: 可以说参考安装文档，安装正确的PaddleSlim和Paddle版本。如果这里固定死，后续文档还得修改。
